### DISCUSSION TITLE
Sign In -- to what?

### DISCUSSION THREAD

**Discussion by seanmceligot** [zed-industries/zed#16832] [Feature Requests]
I'm surprised at how difficult it is to find out what the "Sign In" button is signing in to. I'm sure it's obvious if you've been around a while but not to newcomers.

I've looked through the Zed docs and github project search. 

At first I guessed it was some zed.dev login, but from the code it looks like it's github copilot, but maybe it wants your Github login for checking out projects?

This could be a security issue if users guess wrong and send there username and password meant for one place to another.  Can you just add word or two after Sign In at least and clarify somewhere.

This is arch linux wayland 0.149.5-1. I see "Sign In" in the top right, just in case it's different on other platforms.

**Comment by bajrangCoder** [comment]
The **Sign In** button takes you to sign in with GitHub for access to collaborative features and Zed AI. (Usage limit to prevent abuse.)

**Comment by gouravd** [comment]
Where is this Sign In Button in v0.156 on MacOS?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
